src/styles/_ShoppingBag.scss

```diff
@@ -1,101 +1,187 @@
 @use 'variables' as v;
+@use 'mixins' as m;
 
 .bag {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
+  @include m.center-column;
   background-color: v.$primary-background-color;
   min-height: 100%;
   width: 100%;
   margin: 0;
-}
 
-.bag__items {
-  width: 98%;
-  border-top: v.$secondary-accent-color;
-  border-bottom: v.$secondary-accent-color;
-  border-radius: 3px;
-}
+  .bag__empty {
+    @include m.center-column;
+    background-color: v.$ternary-background-color;
+    width: 98%;
+    text-align: center;
+  }
 
-.bag__summary {
-  display: flex;
-  flex-direction: column;
-  width: 95%;
-}
+  .bag__contents {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    width: 100%;
+  }
 
-.bag__subtotal {
-  display: flex;
-  justify-content: space-between;
-  font-weight: 600;
-}
+  .bag__items,
+  .bag__summary {
+    width: 98%;
+  }
+
+  .bag__summary {
+    display: flex;
+    flex-direction: column;
+    background-color: v.$ternary-background-color;
+    margin: 5px 0;
+  }
+
+  .bag__summary-header {
+    margin: 5%;
+  }
 
-.bag__button {
-  align-self: center;
-  text-decoration: none;
-  background-color: v.$secondary-accent-color;
-  padding: 1rem;
-  border-radius: 30px;
-  color: v.$primary-text-color;
-  margin: 1rem;
+  .bag__subtotal {
+    display: flex;
+    justify-content: space-between;
+    font-weight: 600;
+    margin: 0 5%;
+  }
+
+  .bag__button {
+    align-self: center;
+    text-decoration: none;
+    background-color: v.$secondary-accent-color;
+    padding: 1rem;
+    border-radius: 30px;
+    color: v.$primary-text-color;
+    margin: 1rem;
```

Comment:
The one styling suggestion I have is to put a margin-top: auto to .bag__button so there is not so much negative space at the bottom.  Let me know what you think.

Overall great job styling the breakpoints work great!